tests/org.jboss.tools.central.ui.bot.test/src/org/jboss/tools/central/test/ui/reddeer/ArchetypesTest.java

```diff
@@ -84,7 +127,9 @@ public static void setup() {
 	@After
 	public void teardown() {
 		WorkbenchShellHandler.getInstance().closeAllNonWorbenchShells();
-		for (Project p : new ProjectExplorer().getProjects()) {
+		ProjectExplorer pe = new ProjectExplorer();
+		pe.open();
+		for (Project p : pe.getProjects()) {
 			p.delete(true);
```

Comment:
you can use this: https://github.com/eclipse/reddeer/blob/1b0c5947a3583c632e141f2f7319c1ac2d0f3ade/plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/ui/navigator/resources/AbstractExplorer.java#L141

Reply:
Fixed.